runtime/compiler/trj9/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -15555,120 +15555,71 @@ J9::Z::TreeEvaluator::tabortEvaluator(TR::Node * node, TR::CodeGenerator * cg)
  *
- * 2. Evaluate callNode's children that are needed by the mainline code (with refCount > 2).
- *    OOL-only children evaluation is defered to the OOL path, after the call to
- *    swapInstructionListsWithCompilation()
+ * The steps to evaluate the new BCDCHK node is the following:
  *
- * 3. If applicable, evaluate address node's children (e.g. this is applicable to i2pd but not to PD comparisons)
+ * -# Create a callNode and attached BCDCHK's call parameter children to it. This callNode is to be evaluated
+ *    later in the OOL section
  *
- * 4. Create a handlerLabel that points to the start of the OOL section and replace the second child
- *    with this handlerLabel
+ * -# If applicable, evaluate address node's children (e.g. this is applicable to i2pd but not to PD comparisons)
  *
- * 5. Evaluate the pdopNode (first child) and decrement its refCount.
+ * -# Create a handlerLabel that points to the start of the OOL section
  *
- * 6. Emit a NOP BRC bearing the handlerLabel right after evaluating the pdopNode. This is for SignalHandler.c
+ * -# Evaluate the pdopNode (first child) and decrement its refCount.
  *
- * 7. Undo step 4 by restoring the second child
+ * -# Emit a NOP BRC bearing the handlerLabel right after evaluating the pdopNode. This is for SignalHandler.c
  *
- * 8. Switch to OOL code generation and evaluate the callNode
+ * -# Switch to OOL code generation and evaluate the callNode
  *
- * 9. Evaluate the addressNode (second child of BCDCHK node) to yield a correct address into the byte[]
+ * -# Evaluate the addressNode (second child of BCDCHK node) to yield a correct address into the byte[]
  *
- * 10. Copy the results produced by the call from byte[] back to mainline storage reference
+ * -# Copy the results produced by the call from byte[] back to mainline storage reference
  *
- * 11. Finish up by decRefCount on callNode and addressNode
+ * -# Finish up by decRefCount on callNode and addressNode
  *
- * \param node the BCDCHK node
- * \param pdopNode pdshlOverflow, i2pd or l2pd node
- * \param cg codegen object
- * \param numCallChildre  number of callNode children
- * \param callChildStartIndex  the index of the first callChild under the BCDCHK node
- * \param isResultPD   True if the result of the pdOpNode a PD; false if the result is a binary integer/long
- *                     This also implies that the second node of the BCDCHK node is an address node.
+ * \param node                  the BCDCHK node
+ * \param cg                    codegen object
+ * \param numCallChildre        number of callNode children
+ * \param callChildStartIndex   the index of the first callChild under the BCDCHK node
+ * \param isResultPD            True if the result of the pdOpNode a PD; false if the result is a binary integer/long
+ *                              This also implies that the second node of the BCDCHK node is an address node.
+ * \param isUseVector           If true, emit z14 vector packed decimal instructions
```

Comment:
I don't think we need to mention z14 here as in the future we may use z15 instructions.